src/main/java/taxi/filter/AuthenticationFilter.java

```diff
@@ -0,0 +1,38 @@
+package taxi.filter;
+
+import java.io.IOException;
+import java.util.HashSet;
+import java.util.Set;
+import javax.servlet.Filter;
+import javax.servlet.FilterChain;
+import javax.servlet.FilterConfig;
+import javax.servlet.ServletException;
+import javax.servlet.ServletRequest;
+import javax.servlet.ServletResponse;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import javax.servlet.http.HttpSession;
+
+public class AuthenticationFilter implements Filter {
+    private static final Set<String> URL = new HashSet<>();
```

Comment:
```suggestion
    private Set<String> allowedUrls = new HashSet<>();
```
Don't make it a constant